src/main/java/com/rackspace/ceres/app/config/configValidator/ConcreteGranularityValidator.java

```diff
@@ -0,0 +1,44 @@
+package com.rackspace.ceres.app.config.configValidator;
+
+import com.rackspace.ceres.app.config.DownsampleProperties.Granularity;
+import java.util.List;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class ConcreteGranularityValidator implements ConstraintValidator<GranularityValidator, Object> {
+
+  public void initialize() { }
+
+  @Override
+  public boolean isValid(Object o, ConstraintValidatorContext constraintValidatorContext) {
```

Comment:
Can we take this as `DownsampleProperties` instance instead of `Object`?

Reply:
That brings up a good point that this should actually be a FIELD validator on the `List<Granularity>`